src/envoy/http/jwt_auth/jwt_authenticator.cc

```diff
@@ -22,6 +22,9 @@ namespace Http {
 namespace JwtAuth {
 namespace {
 
+// TODO (lei-tang) remove kJwtPayloadKey after adding the
```

Comment:
can we do this in one PR?  I feel it is related.

Reply:
+1. There is no need to keep TODO. Any place refers to this location should be updated.

Reply:
Will update this PR  to include all places refer to kJwtPayloadKey.

Reply:
remove TODO.  As we discussed,  we are going to keep this sec-istio-auth-userinfo header

Reply:
Done.